apps/wing/src/cli.ts

```diff
@@ -235,6 +235,12 @@ async function main() {
         .preset(3)
         .argParser(parseInt)
     )
+    .addOption(
+      new Option(
+        "-p, --parallel [batch]",
+        "Number of tests to be executed on parallel- if not specified will start executing all at the same time"
```

Comment:
```suggestion
        "-p, --parallel <count>",
        "Number of tests to execute in parallel. 0 to run all at once."
```

Also we could add the default value in here as well

Reply:
zero isn't acceptable in this package- I can add it to our code, or make it behave the same as undefined and run 10 at once, what do you think?

Reply:
Since it's a common option i've seen elsewhere, I think having "0 = no limit" is useful.